.github/workflows/update_news_zenodo.yml

```diff
@@ -0,0 +1,21 @@
+name: Update repo NEWS and zenodo files on approval of pull request
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  build:
+    if: github.event.review.state == 'approved'
```

Comment:
Same remark as above: `and` operations have priority over `or` operations, so for protocols other than `sfp`, the GHA will also run if the review state is other than 'approved' without using parentheses.

Reply:
Good catch, will fix this